### PR TITLE
Presence for Westmanga

### PR DESCRIPTION
**Pull request by mhazm**
Continuing from #3840 

I am fix no imgur link in both `logo` & `thumbnail` and trying fix [ESLint issues](https://i.imgur.com/d5k1C7V.png)
I am trying pull this presence again..

Please help me with an explanation

I added a few things to my presence this time:
- Presence on searching by Genre
- Presence will change depending on the type of manga type being viewed
Like a `Manga`, `Manhua`, `Manhwa`
- Change state presence data on reading manga with `Chapter on reading` of `Chapter end`

_In discussions with the Westmanga community we don't need presence when doing searches_

_Preview :_
[Example for search by genres](https://i.imgur.com/RcLzr1D.png)
[View Manga](https://i.imgur.com/XvxNXPg.png)
[View Manhua](https://i.imgur.com/O59Ky8D.png)
[View Manhwa](https://i.imgur.com/g7eLAKb.png)
[Presence on Reading](https://i.imgur.com/HVmSRY6.png)

### CODE REVIEW

File: websites/W/Westmanga/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "name": "Watambeee",
+    "id": "338418945620967434"
+  },
+  "service": "Westmanga",
+  "description": {
+    "en": "Westmanga is an Indonesian language manga scanlation website with many isekai manga genres",
+    "id": "Westmanga adalah sebuah website scanlation manga berbahasa indonesia dengan banyak genre manga isekai"
+  },
+  "url": "westmanga.info",
+  "version": "1.0.1",
```
Comment (SlowLife1661) [inline websites/W/Westmanga/dist/metadata.json:12]:
```suggestion
  "version": "1.0.0",
```

**Review by SlowLife1661 — CHANGES_REQUESTED** [review]

**Review by Bas950 — DISMISSED** [review]
LGTM

File: websites/W/Westmanga/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "../../../tsconfig.json",
+    "compilerOptions": {
+      "outDir": "./dist/"
+    }
+  }
+
```
Comment (ririxidev) [inline websites/W/Westmanga/tsconfig.json:7]:
```suggestion
```

**Review by ririxidev — CHANGES_REQUESTED** [review]

**Review by SlowLife1661 — APPROVED** [review]

**Comment by mhazm** [comment]
Its okey or not?

**Review by ririxidev — APPROVED** [review]
Yup! Everythins is correct, thank you for contributing to PreMiD!

**Pull request merged**